python/tests/image/test_imageIO.py

```diff
@@ -24,6 +24,7 @@
 from pyspark.sql.types import BinaryType, StringType, StructField, StructType
 
 from sparkdl.image import imageIO
+from pyspark.ml.image import ImageSchema
```

Comment:
ImageSchema should go after sparkdl imports.

Reply:
I left a comment about this, this is because `spark.ml.image` is monkey patched (the image module path is updated when you import `sparkdl`). I really don't like monkey patches and I think this could cause confusion for our users.